Release_Configure

```diff
@@ -48,7 +48,7 @@ fi
 
 # Get the suffix from Git if none was entered.
 if [ "${version}" = "" ]; then
-	version=`git describe --abbrev=7 --long`
+	version=`git describe --abbrev=0`
```

Comment:
Let's hope that this will not erase the hash within the logs also, which is definitely not intended. We just want to change it for the filenames: .iso and .zip

Reply:
This isn't clear indeed; is it possible to give it a trial run before merging?

Reply:
The build hash within the logs and the watermark were not negatively impacted by this change: The githash is still there:
`(ntoskrnl/kd64/kdinit.c:75) ReactOS 0.4.15-x86-dev (Build 20240909-0.4.15-dev-8632-gbc8c7d1) (Commit bc8c7d185d059677c9fe1c112584e00181807d55)`

But there are 2 todos after this PR:
1.) the "i386" generated as the arch should be turned into "x86":
![image](https://github.com/user-attachments/assets/f95f0b85-0a10-411e-9310-ece1d131dad7)

2.) The distance-counter from the last tag is currently missing entirely in the filenames:
currently after this PR:
```
ReactOS-0.4.15-dev-i386-iso.zip               100%  111MB  10.6MB/s   00:10
ReactOS-0.4.15-dev-i386-live.zip              100%   82MB  10.6MB/s   00:07
ReactOS-0.4.15-dev-src.zip                    100%  101MB  10.0MB/s   00:10
```
while it should have been:
```
ReactOS-0.4.15-dev-**8632**-**x86**-iso.zip               100%  111MB  10.6MB/s   00:10
ReactOS-0.4.15-dev-**8632**-**x86**-live.zip              100%   82MB  10.6MB/s   00:07
ReactOS-0.4.15-dev-**8632**-src.zip                    100%  101MB  10.0MB/s   00:10
```